src/gui/tray/NotificationCache.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <set>
+#include <QObject>
+
+namespace OCC {
+
+struct Notification
+{
+    QString title;
+    QString message;
+};
+
+class NotificationCache : public QObject
```

Comment:
@FlexW Are you sure you need that (small, but still some) overhead of deriving from QObject? Unless you are using features like `signals/slots` or others(from QMetaObject), I'd say, this is not necessary.

Reply:
Clearly overkill to inherit from QObject indeed.